src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -46,61 +47,70 @@
            quoted-message
            from
            timestamp]}
-   message-reaction-view?]
-  (when (or (and (seq response-to) quoted-message) last-in-group? pinned-by message-reaction-view?)
+   show-reactions?]
+  (when (or (and (seq response-to) quoted-message) last-in-group? pinned-by (not show-reactions?))
     (let [[primary-name secondary-name] (rf/sub [:contacts/contact-two-names-by-identity from])
           {:keys [ens-verified added?]} (rf/sub [:contacts/contact-by-address from])]
-      [quo/author
-       {:primary-name   primary-name
-        :secondary-name secondary-name
-        :short-chat-key (address/get-shortened-key (or compressed-key from))
-        :time-str       (datetime/timestamp->time timestamp)
-        :contact?       added?
-        :verified?      ens-verified}])))
+      [rn/view {:margin-bottom 8}
+       [quo/author
+        {:primary-name   primary-name
+         :secondary-name secondary-name
+         :short-chat-key (address/get-shortened-key (or compressed-key from))
+         :time-str       (datetime/timestamp->time timestamp)
+         :contact?       added?
+         :verified?      ens-verified}]])))
 
 (defn system-message-content
   [{:keys [content-type quoted-message] :as message-data}]
   (if quoted-message
     [pin/system-message message-data]
     (case content-type
 
-      constants/content-type-system-text           [system.text/text-content message-data]
+      constants/content-type-system-text
```

Comment:
If you can find the time, please change `case` to `cond` or `condp`. `case` is a trap with non-literal values.